editor.html

```diff
@@ -108,7 +108,7 @@ <h2><i class="fa fa-cogs"></i> <strong>{{ title }}</strong></h2>
                     <th>description</th>
                 </tr>
                 {{#snippets}}
-                <tr class="snippet-selection">
+                <tr id="snippet-{{trigger}}" class="action snippet-selection">
```

Comment:
What does the new ID do?

Reply:
The id is what is sent to the metrics, so it lets you know what snippet what snippet was clicked on.

Reply:
Right, thanks for clarifying.